kiwi/defaults.py

```diff
@@ -690,7 +699,8 @@ def get_shim_loader(root_path):
         shim_file_patterns = [
             '/usr/share/efi/*/shim.efi',
             '/usr/lib64/efi/shim.efi',
-            '/boot/efi/EFI/*/shim.efi'
+            '/boot/efi/EFI/*/shim.efi',
+            '/usr/lib/shim/shimx64.efi'
```

Comment:
This should be `/usr/lib/shim/shim*.efi` as the name differs on the architecture.

Reply:
done